managed/services/victoriametrics/scrape_configs.go

```diff
@@ -418,7 +418,7 @@ func scrapeConfigsForMongoDBExporter(s *models.MetricsResolutions, params *scrap
 			"indexstats",
 			"collstats",
 		}
-		if params.pmmAgentVersion != nil && !params.pmmAgentVersion.Less(version.MustParse("2.41.1-0")) {
+		if !params.pmmAgentVersion.Less(version.MustParse("2.41.1-0")) {
```

Comment:
AFAICS, `params.pmmAgentVersion` can be `nil` as per line 390.

Reply:
It can be nil in 390, but there is return in that logic branch, so here it’s safe to assume that it's not nil